src/data/nat/choose/sum.lean

```diff
@@ -103,6 +103,31 @@ begin
   simpa [sum_range_choose_halfway n] using t
 end
 
+lemma four_pow_le_two_mul_add_one_mul_central_binom (n : ℕ) :
+  4 ^ n ≤ (2 * n + 1) * choose (2 * n) n :=
+begin
+  calc 4 ^ n = (1 + 1) ^ (2 * n) : begin
+      rw one_add_one_eq_two,
+      rw pow_mul,
+      suffices h : 2 ^ 2 = 4,
+      rw h,
+      dec_trivial,
+    end
+  ...        = ∑ m in range (2 * n + 1), choose (2 * n) m : by simp [add_pow 1 1 (2 * n)]
+  ...        ≤ ∑ m in range (2 * n + 1), choose (2 * n) (2 * n / 2) : begin
+      apply sum_le_sum,
+      intros i hi,
+      apply choose_le_middle i (2 * n),
+    end
+  ...        = ∑ m in range (2 * n + 1), choose (2 * n) n : by simp
+  ...        = (2 * n + 1) * choose (2 * n) n : begin
+      rw sum_const,
+      rw finset.card_range,
+      simp,
+    end
```

Comment:
And I think this should be true `by simp`. In which case you should be able to move the preceding line, since `simp` can take care of both simultaneously.

Reply:
yep, I will change